### ISSUE TITLE
Last sentence of auto-save text is nonsense

### ISSUE THREAD

**Issue by KyNorthstar** [Supuhstar/Rent-Split-Calculator#79]
It reads:

> If you don't want me to do this, that's fine! I'll only do it if you click the Turn on auto-save! 🍪 button and I won't save anything.

And it should read:

> If you don't want me to do this, that's fine! I'll only do it if you click the Turn on auto-save! 🍪 button.

**Comment by AaronS16** [comment]
Good morning I would love to work this for you if it's still available.

**Comment by yogeswari05** [comment]
Hi , Can I work on this project ?

**Comment by KyNorthstar** [comment]
Yes! @AaronS16 and @yogeswari05 feel free to fork this repo and submit a pull request. No need to ask 😸

– 🧡
